src/pages/Swap/AdvancedSwapMode/InfiniteScroll/InfiniteScroll.component.tsx

```diff
@@ -0,0 +1,76 @@
+import { ChainId, Token } from '@swapr/sdk'
+
+import { default as InfiniteScrollCom } from 'react-infinite-scroll-component'
+
+import { Loader } from '../../../../components/Loader'
+import { AdvancedViewTransaction } from '../../../../services/AdvancedTradingView/advancedTradingView.types'
+import { LoaderContainer, NoDataMessage } from '../AdvancedSwapMode.styles'
+import { Trade } from '../Trade/Trade.component'
+
+interface InfiniteScrollProps {
+  data: AdvancedViewTransaction[]
+  fetchMore: () => Promise<void>
+  hasMore: boolean
+  showTrades: boolean
+  activeCurrencyOption?: Token
+  isLoading: boolean
+  chainId?: ChainId
+  token0: Token
+  isFetched: boolean
+  scrollableTarget: string
+}
+
+export const InfiniteScroll = ({
+  chainId,
+  data,
+  fetchMore,
+  hasMore,
+  isLoading,
+  showTrades,
+  activeCurrencyOption,
+  token0,
+  scrollableTarget,
+  isFetched,
+}: InfiniteScrollProps) => {
+  return (
+    <>
+      <InfiniteScrollCom
+        dataLength={data.length}
+        next={fetchMore}
+        hasMore={hasMore}
+        scrollableTarget={scrollableTarget}
+        loader={null}
+        scrollThreshold={1}
+      >
+        {showTrades &&
+          activeCurrencyOption &&
+          data
+            .sort((firstTrade, secondTrade) => Number(secondTrade.timestamp) - Number(firstTrade.timestamp))
```

Comment:
maybe we can extract this function we pass to sort and give it a descriptive name.

Reply:
.sort((firstTrade, secondTrade) => sortByTimeStamp(firstTrade.timestamp, secondTrade.timestamp))

export const sortByTimeStamp = (timestampFirst: number | string, timestampSecond: number | string) => {
  return Number(timestampSecond) - Number(timestampFirst)
}
 something like that? it is a bit longer but you can reuse, also modify it for order purposes